website/docs/r/s3_bucket.html.markdown

```diff
@@ -309,13 +309,34 @@ resource "aws_s3_bucket" "mybucket" {
 }
 ```
 
+### Using ACL policy grants
+
+```hcl
+data "aws_canonical_user_id" "current_user" {}
+
+resource "aws_s3_bucket" "bucket" {
```

Comment:
Please fix the formatting here. The [`terrafmt` tool](https://github.com/katbyte/terrafmt) can help with this.

Reply:
done